openrtb_ext/request.go

```diff
@@ -20,6 +20,9 @@ const GPIDKey = "gpid"
 // NativeExchangeSpecificLowerBound defines the lower threshold of exchange specific types for native ads. There is no upper bound.
 const NativeExchangeSpecificLowerBound = 500
 
+// TID reserved for Per-Impression Transactions IDs for Multi-Impression Bid Requests.
+const TID = "tid"
```

Comment:
Please name `TIDKey` to match the existing pattern and move to just below "GPIDKey" to keep the special keys grouped together.